extensions/storages/source/.test/.env

```diff
@@ -0,0 +1,4 @@
+S3_ACCESS_KEY=u234
```

Comment:
How did this file bypass `.gitignore`?

Reply:
because it's folder `.test`, not `test`. I removed it